treeherder/model/migrations/0034_job_details_unique_together.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0033_job_detail_field_lengths_decrease'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            # Using ALTER IGNORE so that it will automatically delete duplicates
+            ("ALTER IGNORE TABLE `job_detail` ADD CONSTRAINT "
```

Comment:
I'll be deleting duplicates with a manual SQL command, but this is a fallback in case there are any left, or people have them on their local machine.